### PR TITLE
Updating workflows/transcriptomics/rnaseq-sr from 0.4 to 0.5

### PR DESCRIPTION
**Pull request by planemo-autoupdate**
Hello! This is an automated update of the following workflow: **workflows/transcriptomics/rnaseq-sr**. I created this PR because I think one or more of the component tools are out of date, i.e. there is a newer version available on the ToolShed.

By comparing with the latest versions available on the ToolShed, it seems the following tools are outdated:
* `toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a+galaxy1` should be updated to `toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.10b+galaxy2`

The workflow release number has been updated from 0.4 to 0.5.

### CODE REVIEW

**Comment by lldelisle** [comment]
Hi,
I have 2 issues with this PR:
- In the CI, I get the following error:
```
Unexpected HTTP status code: 400: {"err_msg": "Workflow was not invoked; the following required tools are not installed: pick_value (version 0.1.0)", "err_code": 0, "traceback": "Traceback (most recent call last):\n  File \"/tmp/tmpte_429am/galaxy-dev/lib/galaxy/web/framework/decorators.py\", line 337, in decorator\n    rval = func(self, trans, *args, **kwargs)\n  File \"/tmp/tmpte_429am/galaxy-dev/lib/galaxy/webapps/galaxy/api/workflows.py\", line 751, in invoke\n    raise exceptions.MessageException(missing_tools_message)\ngalaxy.exceptions.MessageException: Workflow was not invoked; the following required tools are not installed: pick_value (version 0.1.0)\n"}

```
- When I try the workflow on usegalaxy.org I have an issue: there are 2 subworkflows and only one of them should be run because they both have conditional and one is the contrary of the other. However, they both run... Any idea, why?
Here is a 'simplified workflow':
image

**Comment by mvdbeek** [comment]
> * In the CI

that'll be fixed by https://github.com/galaxyproject/planemo/pull/1385

> Any idea, why?

No idea, I'll see what's up with that

**Comment by lldelisle** [comment]
I can share the minimum workflow with you if you want.

**Comment by mvdbeek** [comment]
Ah yes, that would be helpful!

**Comment by lldelisle** [comment]
It seems that there is still a bug. Let see that on Monday.

**Comment by mvdbeek** [comment]
Ah, yes, a conditional workflow output, we missed that case!

File: workflows/transcriptomics/rnaseq-sr/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [0.5] 2023-03-17
+
+### Automatic update
+- `toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.8a+galaxy1` was updated to `toolshed.g2.bx.psu.edu/repos/iuc/rgrnastar/rna_star/2.7.10b+galaxy4`
+
+### Manual update
+- Use STAR to compute normalized strand splitted coverage
```
Comment (mvdbeek) [inline workflows/transcriptomics/rnaseq-sr/CHANGELOG.md:9]:
```suggestion
- Use STAR to compute normalized strand-specific coverage
```

File: workflows/transcriptomics/rnaseq-sr/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+- Propose StringTie to compute FPKM etc...
```
Comment (mvdbeek) [inline workflows/transcriptomics/rnaseq-sr/CHANGELOG.md:10]:
```suggestion
- Add option to use StringTie to compute FPKM
```

File: workflows/transcriptomics/rnaseq-sr/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+- Put cufflinks step optional
```
Comment (mvdbeek) [inline workflows/transcriptomics/rnaseq-sr/CHANGELOG.md:11]:
```suggestion
- Make cufflinks step optional
```

**Review by mvdbeek — APPROVED** [review]
Looks good, but can you apply the same language edits from the PE PR ?

**Comment by lldelisle** [comment]
Thanks. I combined both in #211 (was easier for me to introduce spelling changes in both places)

**Pull request closed without merging**